hydra-node/test/Hydra/Chain/Direct/TxSpec.hs

```diff
@@ -259,11 +289,10 @@ genBlueprintTxWithUTxO =
       >>= addSomeReferenceInputs
       >>= addValidityRange
       >>= addRandomMetadata
-      >>= removeRandomInputs
       >>= addCollateralInput
  where
   spendingPubKeyOutput (utxo, txbody) = do
-    utxoToSpend <- (genUTxOAdaOnlyOfSize . getPositive) . Positive =<< choose (0, 50)
+    utxoToSpend <- genUTxOAdaOnlyOfSize =<< choose (0, 3)
```

Comment:
why did we reduce the size?

Reply:
The code was generating up to 50 inputs, then pruned it down to 1 in `removeRandomInputs`. So, in fact, I increased it from 1 to sometimes `3` ;)